transcom/mymove
# kickback B-18648 multiplier was showing up as undefined on non FSC items and mileage didn't show comma

# Agility Story: [B-18648](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18648)

## Test steps I took to recreate
1. Make a HHG move using the HHGNeedsCounseling test harness data.
2. Take note of the move locator and the username of the customer that created the move from the JSON object shown.
3. Navigate back to the Sign In page for customers.
4. Sign in as the user who created the move.
5. Go to the move and add a to an HHG move shown.
6. During the shipment creation process, make sure you enter the following details for Origin and Destination:
- Origin ZIP: 01824
- Origin Duty Location: Chelmsford MA
- Dest ZIP: 62225
- Dest Duty Location: Scott AFB
7. After the shipment has been created, switch to a SC user role under the AGFM and send and approve the move.
8. Switch to TOO and approve the move.
9. Switch to Prime and add the following service items:
- Origin SIT
10. Switch to TOO and approve the new service items
11. Switch back to Prime and choose the "Create payment request" option.
12. Create payment requests for the following:
- FSC
- Origin SIT FSC
- 1st Day Origin SIT
13. Switch back to the TOO or TIO user role and attempt to review the payment requests.
14. On each, show calculations and you should get the following findings:
- Each of the FSC items should show a weight based FSC multiplier with a value
- Mileage value should have a comma in the thousands place
